examples-extra/docs_example/tsconfig.json

```diff
@@ -16,7 +16,7 @@
 
     /* Linting */
     "strict": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Why are we turning this off?

Reply:
Because the generated code now produces unused imports which causes this to fail. Its not a big deal for fully built packages because they get the sdk from npm and so this would never hit. But if you generate in place it does.